James review (#17)
Raphtory/raphtory.github.io: docs/user-guide/persistent-graph/2_ambiguity.md

# Handling of ambiguous updates

> Whereas there is a natural way to construct a _link-stream_ graph regardless of the order the updates come in, throwing deletions into the mix can lead to questions about what graph to create. Let's look at the below example. 

Reviewer: I believe there is some mistakes in the persistent graphs results - we can go through this together

## Order of resolving additions and deletions

{{code_block('getting-started/persistent-graph','behaviour_1',['Graph'])}}

Here two edges between Alice and Bob overlap in time: one starting at time 1 and ending at time 5, another starting at time 3 and ending at time 7. 

For *link-stream* graphs in Raphtory are allowed to have edges between the same pair of nodes happening at the same instant. However, when we look at the exploded edges of this *PersistentGraph* graph, the following is returned:

!!! Output

    ```python exec="on" result="text" session="getting-started/persistent-graph"
    --8<-- "python/getting-started/persistent-graph.py:behaviour_1"
    ```

> Here, we see two edges which are mutually exclusive with each other in time: one from time 1 to time 3 and one from time 3 to time 5. The second deletion at time 7 is ignored. 

Author: ```suggestion
Two edges are created, one that exists from time 1 to time 3 and another that exists from time 3 to time 5. The second deletion at time 7 is ignored. 
```
